src/main/java/org/terasology/manualLabor/systems/ShearingSystem.java

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.manualLabor.systems;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.engine.core.Time;
+import org.terasology.engine.entitySystem.entity.EntityManager;
+import org.terasology.engine.entitySystem.entity.EntityRef;
+import org.terasology.engine.entitySystem.event.ReceiveEvent;
+import org.terasology.engine.entitySystem.systems.BaseComponentSystem;
+import org.terasology.engine.entitySystem.systems.RegisterSystem;
+import org.terasology.engine.logic.characters.events.AttackEvent;
+import org.terasology.engine.logic.delay.DelayManager;
+import org.terasology.engine.logic.delay.PeriodicActionTriggeredEvent;
+import org.terasology.engine.logic.inventory.ItemComponent;
+import org.terasology.engine.registry.In;
+import org.terasology.gestalt.assets.ResourceUrn;
+import org.terasology.manualLabor.components.ShearableComponent;
+import org.terasology.wildAnimals.event.AnimalSpawnEvent;
+
+import java.util.ArrayList;
+
+@RegisterSystem
+public class ShearingSystem extends BaseComponentSystem {
+    public static final int HITS_TO_SHEAR = 5;
+    public static final ArrayList<ItemComponent> SHEARING_ITEMS = null; //TODO yet to decide
+    public static final int HAIR_REGROWTH_TIME = 3*60*1000; // In ms
+    public static final String HAIR_REGROWTH_ACTION_ID = "ManualLabor:HairRegrowthAction";
+    private static final Logger logger = LoggerFactory.getLogger(ShearingSystem.class);
+
+    @In
+    protected Time time;
+
+    @In
+    private DelayManager delayManager;
+
+    @In
+    private EntityManager entityManager;
+
+    @ReceiveEvent
+    public void onAnimalSpawn(AnimalSpawnEvent event, EntityRef entityRef) {
+        if (entityRef.getParentPrefab().getUrn().equals(new ResourceUrn("WildAnimals:Sheep"))) {
+            entityRef.addComponent(new ShearableComponent());
+        }
+    }
```

Comment:
Not entirely sure, tbh, but I _guess_ we could even add a delta prefab adding this component to `WildAnimals:Sheep` :thinking:

Reply:
I think we can wait till the spawning module is created, as this will eventually be changed depending on the mechanism the spawning module will use to spawn animals. This is definitely temporary.

Reply:
not sure how the new spawning module affects this.
I'm assuming that that will, too, use the prefabs provided, for instance, by `WildAnimals` and something like a "Shearable" component should be added using the delta mechanism irrespective of how the spawning works :thinking: 
it's more of a general concept and not specific to how we currently spawn animals.

Reply:
I second that. This should be data-driven as much as possible, and if all the components of a sheep are defined in [`sheep.prefab`](https://github.com/Terasology/WildAnimals/blob/develop/assets/prefabs/animals/sheep.prefab), that's where Sheep's use of ShearableComponent should be specified as well.

The delta system means we can put it places other than _that specific file,_ but it should be something using that mechanism.